src/main/java/Task.java

```diff
@@ -0,0 +1,38 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task (String description){
```

Comment:
I would suggest leaving whitespace between the method arguments and the opening curly bracket such that:
public void someMethod (String description) {
I noticed the same issue in other files as well, so do try to apply it consistently.
(This rule generally applies for other statements such as if-else, while, for, try, etc.)

Reply:
Appreciate that. I have made the changes.